pulpcore/cli/rpm/remote.py

```diff
@@ -44,6 +46,7 @@ def remote(ctx: click.Context, pulp_ctx: PulpContext, remote_type: str) -> None:
         "--policy", type=click.Choice(["immediate", "on_demand", "streamed"], case_sensitive=False)
     ),
     click.option("--sles-auth-token"),
+    click.option("--uln-server-base-url", default="https://linux-update.oracle.com/")
```

Comment:
This should be a `pulp_option`, with a help text and `allowed_with_contexts` specifier.

Reply:
Added the tests and `pulp_option`.